cmake/profile.cmake

```diff
@@ -1,4 +1,31 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-include(${MBED_ROOT}/cmake/profiles/${MBED_PROFILE}.cmake)
+# Set the default build type if none is specified
+if(NOT CMAKE_BUILD_TYPE)
+    set(CMAKE_BUILD_TYPE "Develop" CACHE
+        STRING "The type of build" FORCE)
+endif()
+
+# Mapping CMAKE_BUILD_TYPE into MBED_BUILD_TYPES, as we understand only 3 profiles
```

Comment:
I do not think this comment belongs on this line.
It should probably go after the empty line on line 17 (right before `get_property(multi_config GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)`).

All you are doing there is setting a variable.

Reply:
reviewing now and rebasing to add your 2 suggestion